### PR TITLE
Chore: remove binwrap dependency

### PR DESCRIPTION
**Pull request by academo**
Removes binwrap as a node dependency.

We used to use binwrap to distribute the binary to use it with npx but
it seems binwrap is not maintainted anymore and security issues had been
reported due to sub-dependencies vulnerabilities.

Tihs PR replicates the functionality of binwrap and reduces the node
dependencies to only `tar`

### CODE REVIEW

File: bin/index.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env node
```
Comment (academo) [inline bin/index.js:1]: This script effectively replaces all the work binWrap was doing

**Review by tolzhabayev — APPROVED** [review]
did not test but looks plausible

**Review by xnyo — APPROVED** [review]
LGTM! I have tested it and it works well, awesome work! :rocket:

**Review by jackw — APPROVED** [review]
LGTM. 🚀

**Pull request merged**